doc/user/migration.md

```diff
@@ -0,0 +1,70 @@
+## User / Migrations API
+[Back to the "Users API"](../../users.md) | [Back to the navigation](../../README.md)
+
+# List user migrations
+
+https://docs.github.com/en/rest/migrations/users?apiVersion=2022-11-28#list-user-migrations
+
+```php
+$migrations = $client->api('user')->migration()->list([
+    'per_page' => 10
+]);
```

Comment:
Would not document the per_page example here, we want user to use the `ResultPager` class for this functionality